vms/home/templates/home/home.html

```diff
@@ -16,19 +16,13 @@
 </div>
 
 {% endif %}
-
 <div class="jumbotron jumbotron-custom">
-    <h1 class="index">Volunteer Management System</h1>
-    <p>{% trans "Systers, an Anita Borg Institute Community, has many volunteers that devote their time, skills and commitment to our vision. Volunteer Management System serves as a place where volunteers can view volunteer events, sign up for volunteer jobs and log volunteer hours. Volunteer Management System also enables volunteer managers to manage their volunteers and run volunteer related reports." %}
-    </p>
-    {% if user.is_authenticated %}
```

Comment:
Can I know why was this removed? The issue says just to edit the test so I suppose this button is fine if it is not removed. The number of lines are also decreased which is i guess not leading CodeCov build to pass.